zetaclient/metrics/metrics.go

```diff
@@ -224,6 +224,54 @@ var (
 		},
 		[]string{"client"},
 	)
+
+	InboundObservationsBlockScanTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: ZetaClientNamespace,
+		Name:      "inbound_votes_block_scan_total",
+		Help:      "Total number of inbound observations from block scan",
+	}, []string{"chain"})
+
+	InboundObservationsTrackerTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: ZetaClientNamespace,
+		Name:      "inbound_observations_tracker_total",
```

Comment:
This one always show 0 on Grafana

Reply:
it would show if only if trackers get added , how example thisis from arun from the stress test 
```
zetaclient_inbound_observations_tracker_total{chain="goerli_localnet",internal="true"} 15
```

ill still look into it a bit more ,to see if the metric is working as expected